Realm/RLMSyncManager.h

```diff
@@ -79,14 +79,6 @@ typedef void(^RLMSyncErrorReportingBlock)(NSError *, RLMSyncSession * _Nullable)
 @property (nonatomic, copy) NSString *appID;
 
 /**
- Whether SSL certificate validation should be disabled. SSL certificate validation is ON by default. Setting this
```

Comment:
Would it be okay to keep the API, stick a deprecation notice on it, and have it do nothing?